packages/@lwc/engine-core/src/framework/modules/static-parts.ts

```diff
@@ -105,3 +104,39 @@ export function applyStaticParts(
         applyRefs(part, owner);
     }
 }
+
+/**
+ * Reapplies static parts mounting to newly generated vnodes.
+ *
+ * @param n1 - the previous VStatic vnode
+ * @param n2 - the current VStatic vnode
+ * @param root - the root element
+ * @param renderer - the renderer to use
+ */
+export function patchStaticParts(n1: VStatic, n2: VStatic, root: Element, renderer: RendererAPI) {
+    if (!process.env.IS_BROWSER) {
```

Comment:
Could you remind me of the reason for this? Because we don't do any patching at all on the server?

Reply:
I think it's because in `@lwc/engine-server` we don't support refs or event listeners but TBH I'm using it because it was in `applyStaticParts`.

This might change when we're looking at the attributes/styles/classes though. I'll repeat the [comment](https://github.com/salesforce/lwc/blob/master/packages/%40lwc/engine-core/src/framework/modules/static-parts.ts#L83-L84) from `applyStaticParts` so we can keep track of the reasoning.